File: app/src/main/java/in/completecourse/questionbank/model/NotificationModel.java
package in.completecourse.questionbank.model;

public class NotificationModel {
    String mHeading, mSubHeading, url, serial;

    public String getmHeading() {
        return mHeading;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getmSubHeading() {
        return mSubHeading;
    }

    public void setmHeading(String mHeading) {
        this.mHeading = mHeading;
    }

    public void setmSubHeading(String mSubHeading) {
        this.mSubHeading = mSubHeading;
    }

    public NotificationModel(){

    }

    public String getSerial() {
        return serial;
    }

    public void setSerial(String serial) {
        this.serial = serial;
    }

// --Commented out by Inspection START (2/11/19 12:17 AM):
//    public NotificationModel(String heading, String subheading, String url){
//        this.mHeading = heading;
//        this.mSubHeading = subheading;
//        this.url = url;
//    }
// --Commented out by Inspection STOP (2/11/19 12:17 AM)
}

File: app/src/main/java/in/completecourse/questionbank/model/NotificationModel.kt
package `in`.completecourse.questionbank.model

class NotificationModel {
    var mHeading: String? = null
    var mSubHeading: String? = null
    var url: String? = null

    // --Commented out by Inspection START (2/11/19 12:17 AM):
    var serial: String? = null
    fun getmHeading(): String? {
        return mHeading
    }

    fun getmSubHeading(): String? {
        return mSubHeading
    }

    fun setmHeading(mHeading: String?) {
        this.mHeading = mHeading
    }

    fun setmSubHeading(mSubHeading: String?) {
        this.mSubHeading = mSubHeading
    }
    //    public NotificationModel(String heading, String subheading, String url){
    //        this.mHeading = heading;
    //        this.mSubHeading = subheading;
    //        this.url = url;
    //    }
    // --Commented out by Inspection STOP (2/11/19 12:17 AM)
}